jewelbots_nRF51822_board_v100/cores/JWB_nRF51822/Timer.cpp

```diff
@@ -56,5 +61,10 @@ extern "C"{
     }
   }
 
+  uint32_t Timer::millis(void) {
+    uint32_t num_ticks;
+    app_timer_cnt_get(&num_ticks); // TODO What to do with returned status?
```

Comment:
Here's what I was able to dig up - `app_timer_cnt_get` returns whether it was successful or not. So you could look to see if it equals `NRF_SUCCESS`
```
/**@brief Function for returning the current value of the RTC1 counter.
 *
 * @param[out] p_ticks   Current value of the RTC1 counter.
 *
 * @retval     NRF_SUCCESS   If the counter was successfully read.
 */
uint32_t app_timer_cnt_get(uint32_t * p_ticks);
```

For example, button_handler.c does this:
```
    err_code = app_timer_cnt_get(&button_tick);
    APP_ERROR_CHECK(err_code);
```